torch/onnx/utils.py

```diff
@@ -1660,6 +1669,64 @@ def _export(
     return torch_out
 
 
+@_beartype.beartype
+def _add_onnxscript_fn(
+    model_bytes: bytes,
+    graph: _C.Graph,
```

Comment:
Why do we need the graph as an extra input? Since the onnx model is available, can we not find out which functions are used from the onnx model itself? It seems to me that would help decouple this utility (making it independent how the model is created).

Reply:
Great idea! We should totally use more onnx API.

Reply:
Done